data/flyway/build.gradle

```diff
@@ -1,28 +1,28 @@
-apply plugin: 'java'
-apply plugin: 'application'
+apply plugin: "java"
+apply plugin: "application"
 
 repositories {
     mavenCentral()
 }
 
 dependencies {
-    implementation group: 'org.flywaydb', name: 'flyway-gradle-plugin', version: '8.4.3'
-    implementation(group: 'org.flywaydb', name: 'flyway-commandline', version: '8.4.3') {
-        exclude group: 'com.microsoft.sqlserver', module: 'msql-jdbc'
-        exclude group: 'com.h2database', module: 'h2'
-        exclude group: 'com.pivotal.jdbc', module: 'greenplumdriver'
-        exclude group: 'net.sourceforge.jtds', module: 'jtds'
-	exclude group: 'mysql', module: 'mysql-connector-java'
-        exclude group: 'org.apache.derby', module: 'derby'
-        exclude group: 'org.apache.derby', module: 'derbyclient'
-        exclude group: 'org.hsqldb', module: 'hsqldb'
-        exclude group: 'org.mariadb.jdbc', module: 'mariadb-java-client'
-        exclude group: 'org.xerial', module: 'sqlite-jdbc'
-	exclude group: 'com.google.protobuf', module: 'protobuf-java'
-	exclude group: 'com.google.guava', module: 'guava'
-	constraints {
-	    implementation('org.postgresql:postgresql:42.2.14.jre6')
-	}
+    implementation group: "org.flywaydb", name: "flyway-gradle-plugin", version: "8.5.2"
+    implementation(group: "org.flywaydb", name: "flyway-commandline", version: "8.5.2") {
+        exclude group: "com.microsoft.sqlserver", module: "msql-jdbc"
+        exclude group: "com.h2database", module: "h2"
+        exclude group: "com.pivotal.jdbc", module: "greenplumdriver"
+        exclude group: "net.sourceforge.jtds", module: "jtds"
+	    exclude group: "mysql", module: "mysql-connector-java"
+        exclude group: "org.apache.derby", module: "derby"
+        exclude group: "org.apache.derby", module: "derbyclient"
+        exclude group: "org.hsqldb", module: "hsqldb"
+        exclude group: "org.mariadb.jdbc", module: "mariadb-java-client"
+        exclude group: "org.xerial", module: "sqlite-jdbc"
+        exclude group: "com.google.protobuf", module: "protobuf-java"
+        exclude group: "com.google.guava", module: "guava"
+        constraints {
+	       implementation("org.postgresql:postgresql:42.2.14.jre6")
```

Comment:
@fec-jli  postgresql:42.2.14.jre6 is not available in flyway-8.5.2. The postgresql jar  available in flyway-8.5.2 is postgresql-42.2.25.jar. Can you replace it?

Reply:
Here is the screenshot from [test/upgrade_flyway852 circle build](https://app.circleci.com/pipelines/github/fecgov/openFEC?branch=test%2Fupgrade_flyway852&filter=all)
![Screen Shot 2022-03-08 at 9 22 11 PM](https://user-images.githubusercontent.com/11650355/157360157-1248a3d3-cc30-4a2a-8e9e-a7daae29a278.png)